app/assets/stylesheets/_helper_classes.scss

```diff
@@ -54,3 +65,15 @@ label[for$=_unknown] {
 .text-error {
   color: red;
 }
+
+.pull-right-content {
+  float: right;
+}
+
+.complete-label {
```

Comment:
do these need to be in the top-level namespace for our CSS? Maybe it's time to have the BEM conversation